saidump/saidump.cpp

```diff
@@ -10,18 +13,26 @@ extern "C" {
 #include "swss/table.h"
 #include "meta/sai_serialize.h"
 #include "sairediscommon.h"
+#include "swss/json.h"
+#include "swss/json.hpp"
 
 #include <getopt.h>
 
 // TODO split to multiple cpp
 
 using namespace swss;
+using namespace std;
```

Comment:
remove std, use direct mapping std::string etc

Reply:
Fix it.